### DISCUSSION TITLE
첫번째 모임 후기 스레드

### DISCUSSION THREAD

**Discussion by seriousran** [Pseudo-Lab/growth-collector#5] [모임 후기!]
첫 모임이라 많이 어색했지만, 새로운 성장 가능성을 볼 수 있는 대화들을 나눴습니다!
서로를 알아가고, 우리의 활동을 소개하며, 조금씩 구체화해볼 수 있었던 것 같아요!
성장을 수집하고, 서로의 목표를 향해 성장해나가는 우리들이 되길 바래봅니다!

참가자: 찬란, 창엽, 영승, 현지, 윤희

숙제?!
1. 모임 소감 남기기 (여기!)
2. OKR 만들어 공유하기 ([저기!](https://github.com/Pseudo-Lab/growth-collector/discussions/categories/%EC%9A%B0%EB%A6%AC%EB%93%A4%EC%9D%98-%EC%84%B1%EC%9E%A5-%EB%AA%A9%ED%91%9C))
3. AI 대가, 전문가 리스트업! / 해외 10명, 국내 10명 이상 정도..? ([저기!](https://github.com/Pseudo-Lab/growth-collector/blob/main/list.md))

ps. 사진을 못찍어서 너무 아쉬워요! 다음엔 꼭... 과정마다 자주 찍기로...!

**Comment by soncy0311** [comment]
첫 모임부터 늦은 시간까지 유익한 시간이었습니다.
서로에게 좋은 영향을 받아서 성장하는 기회가 되었으면 좋겠습니다! 😁

**Comment by ZeVicTech** [comment]
첫번째 성장 수집가 모임임에도 불구하고, AI에 관련된 다양한 의견을 나눌 수 있어서 많은 도움이 됐습니다.
또한 OKR 목표를 간략하게 나마 세우고 서로 공유하면서, 성장을 위해 현재 해야할 노력을 좀 더 명확하게 인지할 수 있었습니다.
앞으로도 모임에 꾸준히 참여해서 서로 성장할 수 있도록 노력하겠습니다!

**Comment by yunhee1** [comment]
성장하고 싶은 사람들이 모일 수 있는 자리를 만들어주셔서 감사합니다.
처음 알게 된 분들과 유익한 이야기를 나눌 수 있어 행복했습니다.

말씀해주신 모든 것들이 좋았지만 마지막 남은 25분을 특히나 더 알차게 사용하여 서로의 OKR을 간략하게나마 확인하고, 피드백까지 나누며 꽉꽉 채워 활용할 수 있었던 게 가장 인상깊습니다.

다음 밋업도 너무나 기대가 됩니다!💪

**Comment by shinhyunji36** [comment]
지금까지 막연하게 큰 목표만 주로 세웠었는데, 그 큰 목표를 달성하기 위한 지표인 OKR에 대해서 처음으로 깊게 고민해봤던 자리였습니다! :)
OKR에 대해 처음 설명을 들어서인지 원하는 바들을 어떻게 3개월 단위의 달성 목표를 가진 OKR이라는 지표에 녹여내야할지 감을 잡는 것이 조금 힘들었었는데, 마지막 25분 동안 다른 분들의 OKR 청사진에 대해 듣다보니 약간은  어떻게 설정해야겠다 감을 잡을 수 있던 것 같습니다. 
첫 만남인데 찬란님이 많이 고민해오셨다는 것이 느껴졌고, 앞으로 꾸준히 서로의 성장에 긍정적인 영향을 주고 성장해나가는 과정들을 수집해가면 좋겠습니다!🥰
